### DISCUSSION TITLE
Broken Transport Error with Quatrix.it

### DISCUSSION THREAD

**Discussion by DJCWinyard** [iterate-ch/cyberduck#14738] [Q&A]
Hello,

Very recently, CyberDuck has stopped working with the following SFTP server:

sftp22.rspca.quatrix.it. (port 22)

I have the latest version of CyberDuck for MacOS (version 8.5.9 (39636)). You do not need any login credentials for this server to reproduce this bug - just try connecting to it with CyberDuck - and it reports:

"Broken transport; encountered EOF. The connection attempt was rejected. The server may be down, or your network may not be properly configured."

It is possible to connect to this same server using both the built-in command line sftp client in the terminal on MacOS, and also with FileZilla.

**Comment by ylangisc** [comment]
I can reproduce the issue you see but as the server is closing the connection during handshake it would be helpful to get more information about the server OS and the OpenSSH version.
